leo/commands/init.rs

```diff
@@ -14,58 +14,52 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{
-    cli::*,
-    cli_types::*,
-    errors::{CLIError, InitError},
-};
+use crate::{commands::Command, context::Context};
+use anyhow::{anyhow, Result};
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates